api/authorizers/site.js

```diff
@@ -32,13 +33,30 @@ const authorizeAdmin = (user, site) => (
       // attempts to fetch the repo but it no longer exists and receives a 404
         return Promise.resolve(site.id);
       }
-      return Promise.reject({
+      throw {
         message: siteErrors.ADMIN_ACCESS_REQUIRED,
         status: 403,
-      });
+      };
     })
 );
 
+const authorizeFederalistUsersAdmin = (user) =>
+  FederalistUsersHelper.federalistUsersAdmins(user.githubAccessToken)
+    .then((admins) => {
+      if (!admins.includes(user.username)) {
```

Comment:
not a big deal, but this is a bit repetitive, you could just throw `new Error()` here since it will be caught in the next `catch` and throw the error with the message there, or make a function?

Reply:
removed! 👍